### ISSUE TITLE
library not found for -lGoogleToolboxForMac

### ISSUE THREAD

**Issue by hjJunior** [evollu/react-native-fcm#864]
1. What version of RN and react-native-fcm are you running?
`react-native: 0..54.0`
`react-native-fcm: ^14.1.3`
2. What device are you using? (e.g iOS9 emulator, Android 6 device)?
not applicable
3. Is your app running in foreground, background or not running?
not applicable

I'm getting error on build saying that library not found, this have done just after I use pod install

## My PodFile

```
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

#pod 'Firebase/Core'
pod 'Firebase/Messaging'
#pod 'react-native-fcm', :path => '../node_modules/react-native-fcm'

target 'MyApp' do
  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
  # use_frameworks!

  # Pods for MyApp

  target 'MyApp-tvOSTests' do
    inherit! :search_paths
    # Pods for testing
  end

  target 'TabelionatoAmorimTests' do
    inherit! :search_paths
    # Pods for testing
  end

end
```

I've tried to uncomment the following lines and remove and installing again running pod install
```
#pod 'Firebase/Core'
#pod 'react-native-fcm', :path => '../node_modules/react-native-fcm'
```

**Comment by evollu** [comment]
How do you integrate with react-native-fcm?
manual linking (the result is you have a project in main project -> Library folder)
react-native link (the result is you have a project under Pod project)

**Comment by hjJunior** [comment]
Using `react-native link` and pod install

**Comment by evollu** [comment]
can you provide an example project with the issue?

**Comment by hjJunior** [comment]
Yep, do want that I create a repository here in GitHub or maybe a private repository in Bitbucket, I mean what's the best way ?

**Comment by evollu** [comment]
public git repo

**Comment by hjJunior** [comment]
Okay, I'll init a new repo, with new code, and I back to here to put the link ;)

**Comment by monkart** [comment]
Hi! I have the same problem

**Comment by hjJunior** [comment]
I even tried to clone the example repository, but I'm getting the same error

**Comment by hjJunior** [comment]
Any update?

**Comment by evollu** [comment]
looking

**Comment by evollu** [comment]
tried the repo and build without issue.
I did
1. pod repo update
2. pod install
opened a PR against yours because my project file gets changed

**Comment by marciovsena** [comment]
Hi! I have the same problem
